src/applications/financial-status-report/components/resolution/ResolutionExplainerWidget.jsx

```diff
@@ -41,9 +45,8 @@ const ResolutionExplainerWidget = ({
           close-btn-aria-label="Close notification"
           disable-analytics="false"
           full-width="false"
-          show-icon
```

Comment:
Has `show-icon` been deprecated, or is it not necessary anymore?

Reply:
Yeah I'm pretty sure the icons are now controlled by the status prop. 

As far as I can tell show-icon was removed from the v3 and the V1 component props

https://design.va.gov/storybook/?path=/docs/uswds-va-alert